tests/test_aliases.py

```diff
@@ -16,19 +16,24 @@
         ("dg.3456/test_get_aliases", ["1", "2", "3", "4", "5"], False),
         ("dg.4567/test_get_aliases", [], True),
         ("dg.4567/test_get_aliases", [], False),
-        ("dg.5678/test_get_aliases", ["dg.1234/test_get_aliases"], True),
-        ("dg.5678/test_get_aliases", ["dg.1234/test_get_aliases"], False),
-        ("dg.6789/test_get_aliases", ["!@(#_*&$)^-!@#)%*_(&"], True),
-        ("dg.6789/test_get_aliases", ["!@(#_*&$)^-!@#)%*_(&"], False),
-        ("dg.7890/test_get_aliases", ["/\\|_.,-;__"], True),
-        ("dg.7890/test_get_aliases", ["/\\|_.,-;__"], False),
+        ("dg.5678/test_get_aliases", None, True),
+        ("dg.5678/test_get_aliases", None, False),
+        ("dg.6789/test_get_aliases", ["dg.1234/test_get_aliases"], True),
+        ("dg.6789/test_get_aliases", ["dg.1234/test_get_aliases"], False),
+        ("dg.7890/test_get_aliases", ["!@(#_*&$)^-!@#)%*_(&"], True),
+        ("dg.7890/test_get_aliases", ["!@(#_*&$)^-!@#)%*_(&"], False),
+        ("dg.8890/test_get_aliases", ["/\\|_.,-;__"], True),
+        ("dg.8890/test_get_aliases", ["/\\|_.,-;__"], False),
     ],
 )
 def test_create_read_delete_new_aliases(guid, aliases, client, is_post):
     """
     Create a metadata record, then try POST and PUT new aliases.
     Ensure you are able to GET the new aliases and DELETE them one by one.
     """
+    # convert None to empty list
+    aliases = aliases or []
```

Comment:
isn't the point of testing None that MDS should handle it, not the test?

Reply:
omg. whoops, I meant for this to just be used to compare the output not also as the input. I'll fix it. To be clear, the **output** should be handled by the code and it should return an empty list. The expected behavior is that it always returns a list, no matter the input.